### ISSUE TITLE
What's meaning of wh_offset_base ?

### ISSUE THREAD

**Issue by HAOCHENYE** [ZJULearning/ttfnet#34]
It seems that the model will converge much more slower if **wh_offset_base =1**, especially the wh_loss. Besides, compared with centernet, the loss weight of wh_loss is much more larger than hm_loss（centernet: wh_loss_weight=0.1, hm_loss=1),why?

**Comment by PeterVennerstrom** [comment]
The wh_offset_base enables smaller, more easily predicted logits for wh.  Wh logits are more similar to hm logits using this method.  

The loss values after weighting are balanced for wh and hm.  TTFNet uses a different wh loss than centernet requiring a different loss re-weighting to keep the two losses in approximate balance.

**Comment by HAOCHENYE** [comment]
Thanks for your patience!
